rero_ils/modules/holdings/api_views.py

```diff
@@ -139,3 +149,99 @@ def receive_regular_issue(holding_pid):
         abort(400)
     # the created item of type issue is returned
     return jsonify({'issue': issue})
+
+
+def do_holding_jsonify_action(func):
+    """Jsonify loan actions for holdings methods.
+
+    This method for the circulation actions that required access to the holding
+    object before executing the invenio-circulation logic.
+    """
+    @wraps(func)
+    def decorated_view(*args, **kwargs):
+        try:
+            data = flask_request.get_json()
+            description = data.pop('description')
```

Comment:
place the `try...except KeyError` only around this line.
include the `raise MissingRequiredParameterError` into the except block
If a KeyError is fired by another part of your code, It will be catched by last `except Exception` block.